resources/monitoring/charts/prometheus-istio/values.yaml

```diff
@@ -47,8 +47,6 @@ alertmanager:
   ## alertmanager container image
   ##
   image:
-    repository: prom/alertmanager
```

Comment:
we are not using that image/functionality, please just comment out and do not specify in the listing. Please revert related changes of defining the image to have it more easy to update the chart

Reply:
The image was restored

Reply:
The image was restored